src/azure-cli/azure/cli/command_modules/redis/_help.py

```diff
@@ -216,3 +229,73 @@
 type: command
 short-summary: Show the identity assigned for Azure cache for Redis.
 """
+
+helps['redis access-policy'] = """
+type: group
+short-summary: Manage access policies for Redis Cache
+"""
+
+helps['redis access-policy create'] = """
+type: command
+short-summary: Adds an access policy to the Redis Cache
+long-summary: Usage example - az redis access-policy create -g testResourceGroup -n testCacheName --access-policy-name testAccessPolicy --permissions "+get +hget"
```

Comment:
```suggestion
examples:
  - name: blablabla
    text: az redis access-policy create -g testResourceGroup -n testCacheName --access-policy-name testAccessPolicy --permissions "+get +hget"
```
Please use `examples` definition instead of `long-summary: Usage example ...`

Reply:
Using examples instead of long-summary